runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java

```diff
@@ -60,26 +71,35 @@
 	private final OpponentInfoConfig opponentInfoConfig;
 	private final HiscoreManager hiscoreManager;
 	private final NPCManager npcManager;
+	private final ClientThread clientThread;
 
 	private Integer lastMaxHealth;
 	private int lastRatio = 0;
 	private int lastHealthScale = 0;
 	private String opponentName;
+	private int raidOpponentHealth = 0;
+	private int raidOpponentMaxHealth = 0;
+	private String raidOpponentName;
+	private final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
+	boolean useVanilla = false;
+	boolean shouldUnHide = false;
```

Comment:
boolean fields default to `false`, you don't need to explicitly set that starting value.
```suggestion
	boolean useVanilla;
	boolean shouldUnHide;
```